### PR TITLE
Generate mapping file

### PR DESCRIPTION
**Pull request by Yueqiao12Zhang**
This PR creates a new script that generates an empty mapping file with all the necessary keys. 
Since all headers are used as predicates in the RDF triples, they need to be reconciled to some other URIs. This mapping file will contain a dictionary with all header cells as keys, and all values are empty. Users should reconcile them according to Wikidata / schema.org.

### CODE REVIEW

File: csv2rdf/get_relations.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import csv
```
Comment (dchiller) [inline csv2rdf/get_relations.py:1]: This needs a docstring to explain usage...

File: csv2rdf/get_relations.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    with open(filename, "r") as csv_file:
+        csv_reader = csv.reader(csv_file)
+        header = next(csv_reader)
+        
+    header.append("entity_type")
+
+    for item in header:
+        dt[item] = ""
+        
+    dt["entity_type"] = []
```
Comment (dchiller) [inline csv2rdf/get_relations.py:19]: You don't need to do this with every loop.

**Comment by dchiller** [comment]
@Yueqiao12Zhang See comments above. I have also been lax about commit messages thus far, but going forward please try to be more descriptive. See, for example, the document that Ich sent out about commit messages earlier this week. If you have multiple commits in a row with the same message, that is probably a sign that your messages need to be more specific!

**Comment by Yueqiao12Zhang** [comment]
> @Yueqiao12Zhang See comments above. I have also been lax about commit messages thus far, but going forward please try to be more descriptive. See, for example, the document that Ich sent out about commit messages earlier this week. If you have multiple commits in a row with the same message, that is probably a sign that your messages need to be more specific!
> 
> If you were to solve my two comments in separate commits, they could be something like: `docs: add module docstring to csv2rdf/get_relations.py` and `refactor: stop setting entity_type for every file in csv2rdf/get_relations.py`

What if I do both things in one commit?

**Comment by dchiller** [comment]
> What if I do both things in one commit?

Then write a commit message that describes both? All I'm saying is that "Update [filename]" is never a good commit message because it is too broad. There's always something more specific.

File: csv2rdf/get_relations.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+output_name = "mapping.json"
+dt = {}
+dt["entity_type"] = []
+
+for filename in filenames:
+    with open(os.path.abspath(filename), "r") as csv_file:
+        csv_reader = csv.reader(csv_file)
+        header = next(csv_reader)
+        
+    header.append("entity_type")
```
Comment (dchiller) [inline csv2rdf/get_relations.py:24]: You also don't need to do this in the loop?

**Comment by dchiller** [comment]
Are you using `black` and `pylint`? There are multiple things in this file that those are flagging for me!

File: csv2rdf/get_relations.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import sys
+import os
+
+filenames = sys.argv[1:]
+output_name = "mapping.json"
+dt = {}
+dt["entity_type"] = []
+
+for filename in filenames:
+    with open(os.path.abspath(filename), "r") as csv_file:
```
Comment (dchiller) [inline csv2rdf/get_relations.py:20]: Why do you need the abspath call here?

Comment (Yueqiao12Zhang) [reply csv2rdf/get_relations.py:20]: If we move the test files elsewhere (a test folder inside this directory) and do not use the abspath call, then Python is not able to find the file.

**Review by dchiller — APPROVED** [review]

**Pull request merged**